src/zope/traversing/browser/interfaces.py

```diff
@@ -25,16 +25,16 @@ class IAbsoluteURL(Interface):
     """
 
     def __unicode__():
-        """Returns the URL as a unicode string."""
+        """Return the URL as unquoted str."""
```

Comment:
Yeah this is weird, but fixing it is out of scope for this PR.

(I don't think the "Config with pure python template" PR description is accurate for a PR that drops support for various Python versions.)